posthog/hogql/database/schema/util/where_clause_extractor.py

```diff
@@ -159,23 +174,33 @@ def visit_call(self, node: ast.Call) -> ast.Expr:
             return self.visit_compare_operation(
                 ast.CompareOperation(op=CompareOperationOp.Eq, left=node.args[0], right=node.args[1])
             )
+        elif node.name == "notEquals":
+            return self.visit_compare_operation(
+                ast.CompareOperation(op=CompareOperationOp.NotEq, left=node.args[0], right=node.args[1]),
+            )
         elif node.name == "like":
             return self.visit_compare_operation(
                 ast.CompareOperation(op=CompareOperationOp.Like, left=node.args[0], right=node.args[1])
             )
         elif node.name == "notLike":
             return self.visit_compare_operation(
-                ast.CompareOperation(op=CompareOperationOp.NotLike, left=node.args[0], right=node.args[1])
+                ast.CompareOperation(op=CompareOperationOp.NotLike, left=node.args[0], right=node.args[1]),
             )
         elif node.name == "ilike":
             return self.visit_compare_operation(
                 ast.CompareOperation(op=CompareOperationOp.ILike, left=node.args[0], right=node.args[1])
             )
         elif node.name == "notIlike":
+            return self.visit_compare_operation(
+                ast.CompareOperation(op=CompareOperationOp.NotILike, left=node.args[0], right=node.args[1]),
+            )
+        elif node.name == "in":
             return self.visit_compare_operation(
                 ast.CompareOperation(op=CompareOperationOp.NotILike, left=node.args[0], right=node.args[1])
             )
```

Comment:
Because of the change on the right hand side, shouldn't this now be:

```suggestion
        elif node.name == "in":
            return self.visit_compare_operation(
                ast.CompareOperation(op=CompareOperationOp.In, left=node.args[0], right=node.args[1])
            )
```

Also, seems like `notIn` should also be covered here, unless that happens somewhere else? I know that [`HOGQL_COMPARISON_MAPPING`](https://github.com/PostHog/posthog/blob/9f60b9cdf8a07f0b395c60f521d00498061240bb/posthog/hogql/functions/mapping.py#L93-L106) is used for similar purposes elsewhere — maybe that can just be reused here to simplify and standardize this long conditional?